setfps.lua

```diff
@@ -15,6 +15,10 @@ Usage::
 local cap = ...
 local capnum = tonumber(cap)
 
+if not cap or cap=='-help' then
```

Comment:
I'm not sure what the standard should be, but some scripts use two dashes for "long" parameters and accept "short" parameters with one dash. E.g. `-h` (one dash) and `--help` (two dashes).

Reply:
I've often just used `cap:match('help')` to catch everything.

Reply:
Happy to change that - just went by the few other DFhack scripts I checked against. Typically the standard is `-h`/`--help` as @myk002 stated, but for instance [assign-skills](https://github.com/DFHack/scripts/blob/c44df3b0dfdc35230f91ff2d95a619d69ac88d84/assign-skills.lua#L41) only responds to `-help`.

```suggestion
if not cap or cap:match('help') then
```


Alternately, we could just combine it with the `if not capnumber` line, so anything that doesn't resolve to a number triggers the usage text - not sure it's useful to return the text if we're not going to do anything with it.

Reply:
Yeah, that's what I was thinking. Your alternative is fine with me, assuming that means combining the messages into something like this:
```
Invalid FPS cap value: <val>
Usage: setfps <number>
```
The original issue here is that `..cap` (line 23 currently) fails for things that can't be converted implicitly to strings, e.g. `nil`. Changing that to `..tostring(cap)` should work, though (maybe `tostring(cap or '')` to avoid `nil` showing up in the output).